custom_components/hubitat/hubitatmaker/const.py

```diff
@@ -33,40 +34,98 @@ class DeviceCapability(StrEnum):
     TEMPERATURE_MEASUREMENT = "TemperatureMeasurement"
     THERMOSTAT = "Thermostat"
     WINDOW_SHADE = "WindowShade"
+    WINDOW_BLIND = "WindowBlind"
     FAN_CONTROL = "FanControl"
+    CURRENT_METER = "CurrentMeter"
+    ACCELERATION_SENSOR = "AccelerationSensor"
+    BATTERY = "battery"
+    BULB = "Bulb"
+    CHIME = "Chime"
+    INDICATOR = "Indicator"
+    OUTLET = "Outlet"
+    RELAY_SWITCH = "RelaySwitch"
+    SHOCK_SENSOR = "ShockSensor"
+    SIGNAL_STRENGTH = "SignalStrength"
+    SLEEP_SENSOR = "SleepSensor"
+    SMOKE_DETECTOR = "SmokeDetector"
+    SOUND_PRESSURE_LEVEL = "SoundPressureLevel"
+    SOUND_SENSOR = "SoundSensor"
+    STEP_SENSOR = "StepSensor"
+    TAMPER_ALERT = "TamperAlert"
+    THERMOSTAT_COOLING_SETPOINT = "ThermostatCoolingSetpoint"
+    THERMOSTAT_FAN_MODE = "ThermostatFanMode"
+    THERMOSTAT_HEATING_SETPOINT = "ThermostatHeatingSetpoint"
+    THERMOSTAT_SETPOINT = "ThermostatSetpoint"
+    TIMED_SESSION = "TimedSession"
+    TOUCH_SENSOR = "TouchSensor"
+    UV = "UltravioletIndex"
+    VALVE = "Valve"
+    VIDEO_CAMERA = "VideoCamera"
+    VOLTAGE_MEASUREMENT = "VoltageMeasurement"
+    WATER_SENSOR = "WaterSensor"
+    CONSUMABLE = "consumable"
+    CHANGE_LEVEL = "ChangeLevel"
+    LIGHT_EFFECTS = "LightEffects"
+    MOMENTARY = "Momentary"
+    PUSH = "push"
```

Comment:
Is "push" a valid capability?

Reply:
My bad, this was more for the DeviceCommand section :-) 
https://docs2.hubitat.com/en/developer/driver/capability-list#pushablebutton